src/ray/core_worker/lib/java/org_ray_runtime_WorkerContext.cc

```diff
@@ -0,0 +1,129 @@
+#include "ray/core_worker/lib/java/org_ray_runtime_WorkerContext.h"
+#include <jni.h>
+#include "ray/common/id.h"
+#include "ray/core_worker/context.h"
+#include "ray/core_worker/lib/java/jni_helper.h"
+
+inline ray::WorkerContext *GetWorkerContext(jlong nativeWorkerContext) {
+  return reinterpret_cast<ray::WorkerContext *>(nativeWorkerContext);
+}
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*
+ * Class:     org_ray_runtime_WorkerContext
+ * Method:    nativeCreateWorkerContext
+ * Signature: (I[B)J
+ */
+JNIEXPORT jlong JNICALL Java_org_ray_runtime_WorkerContext_nativeCreateWorkerContext(
+    JNIEnv *env, jclass, jint workerType, jbyteArray jobId) {
+  return reinterpret_cast<jlong>(
+      new ray::WorkerContext(static_cast<ray::rpc::WorkerType>(workerType),
+                             JavaByteArrayToUniqueId<ray::JobID>(env, jobId)));
+}
+
+/*
+ * Class:     org_ray_runtime_WorkerContext
+ * Method:    nativeGetCurrentTaskId
+ * Signature: (J)[B
+ */
+JNIEXPORT jbyteArray JNICALL Java_org_ray_runtime_WorkerContext_nativeGetCurrentTaskId(
+    JNIEnv *env, jclass, jlong nativeWorkerContext) {
+  auto task_id = GetWorkerContext(nativeWorkerContext)->GetCurrentTaskID();
+  return UniqueIDToJavaByteArray<ray::TaskID>(env, task_id);
```

Comment:
It's better to rename this to `IDToJavaByteArray` because `TaskID` is not an `UniqueID`.

Reply:
should be `IdToJavaByteArray` by the way

Reply:
Done.